src/templates/conf.template.py

```diff
@@ -13,7 +13,7 @@
 
 html_title = "{{ theme.display }}"
 
-sys.path.insert(0, os.path.abspath("sample-docs/kitchen-sink/"))
+sys.path.insert(0, os.path.abspath("../../sample-docs/kitchen-sink/"))
```

Comment:
```suggestion
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "kitchen-sink"))
```

Reply:
Actually, that's not the right path it seems... https://github.com/sphinx-themes/sphinx-themes.org/actions/runs/16562065003/job/46833584133?pr=179#step:4:638